mkdocs.yml

```diff
@@ -24,35 +24,37 @@ markdown_extensions:
 extra_javascript: ['https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML','mathjaxhelper.js']
 
 pages:
- - Meep.md
- - Download.md
- - Release_Notes.md
- - FAQ.md
- - Parallel_Meep.md
- - Introduction.md
- - Installation.md
- - Scheme_Tutorial.md
- - Scheme_User_Interface.md
- - C++_Tutorial.md
- - C++_User_Interface.md
- - Materials.md
- - Exploiting_Symmetry.md
- - Perfectly_Matched_Layer.md
- - Synchronizing_the_Magnetic_and_Electric_Fields.md
- - Units_and_Nonlinearity.md
- - Yee_Lattice.md
- - The_Run_Function_Is_Not_A_Loop.md
- - Guile_and_Scheme_Information.md
- - Cylindrical_Coordinates.md
- - Field_Function_Examples.md
- - Acknowledgements.md
- - License_and_Copyright.md
- - Scheme_Tutorials/Ring_Resonator_in_Cylindrical_Coordinates.md
- - Scheme_Tutorials/Resonant_Modes_and_Transmission_in_a_Waveguide_Cavity.md
- - Scheme_Tutorials/Third_Harmonic_Generation.md
- - Scheme_Tutorials/Material_Dispersion.md
- - Scheme_Tutorials/Casimir_Forces.md
- - Scheme_Tutorials/Local_Density_of_States.md
- - Scheme_Tutorials/Optical_Forces.md
- - Scheme_Tutorials/Near_to_Far_Field_Spectra.md
- - Scheme_Tutorials/Multilevel_Atomic_Susceptibility.md
+- Download:        'Download.md'
+- Release Notes:   'Release_Notes.md'
+- FAQ:             'FAQ.md'
+- Meep manual:     'index.md'
+- Introduction:
```

Comment:
Maybe better to organize into "features", "installation", "scheme interface", "c++ interface" sections

And put all the scheme stuff together